### ISSUE TITLE
Invalid XML in test case for cache configuration

### ISSUE THREAD

**Issue by ahus1** [keycloak/keycloak#28681]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

operator

### Describe the bug

There is an unstable operator test https://github.com/keycloak/keycloak/actions/runs/8664622019/job/23761668830?pr=28502 in CacheTest

```
tag name </cache-contianer> must be the same as start tag <cache-container> from line 6 (position: TEXT seen ...</local-cache>\n    </cache-contianer>... @10:23) 
```

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

Test should be stable

### Actual behavior

Test is failing

### How to Reproduce?

Invalid XML in log

**Comment by shawkins** [comment]
Should this just be checked in under the existing issue?

**Comment by ahus1** [comment]
@shawkins - as long as we can cleanly handle the backport, using the same issue works for me.

**Comment by shawkins** [comment]
Yes this was just merged today, I just put the backports into draft so that the fix will be applied there as well before pulling them out of draft.

**Comment by ahus1** [comment]
Sure, then let's do it that way.

**Comment by ahus1** [comment]
Resolved via #28682
